src/components/testCaseLanding/qdm/TestCaseCoverage/CoverageTabs/CoverageTabList.tsx

```diff
@@ -1,110 +1,115 @@
 interface Props {
-  populationCriteria: any;
-  mappedCql: CoverageMappedCql;
   calculationOutput: CqmExecutionResultsByPatient;
+  testCases: any;
+  testCaseGroups: any;
+  cqlDefinitionCallstack: any;
+  groupCoverageResult: any;
+  populationCriteria: any;
+  measureGroups: any;
 }
 
 const allDefinitions = ["Used", "Functions", "Unused"];
 
 const CoverageTabList = ({
+  measureGroups,
+  groupCoverageResult,
   populationCriteria,
-  mappedCql,
-  calculationOutput,
+  cqlDefinitionCallstack,
+  testCaseGroups,
 }: Props) => {
-  const getDefintionCategoryFilteringCondition = (
-    definitionType,
-    statementResult,
-    definitionName
-  ) => {
-    if (definitionType === "Used") {
-      return statementResult[definitionName].relevance !== "NA";
-    }
-    if (definitionType === "Unused") {
-      return statementResult[definitionName].relevance === "NA";
+  const [relevantPops, setRelevantPops] = useState([]);
+  useEffect(() => {
+    if (testCaseGroups && populationCriteria.id) {
+      const selectedGroup = testCaseGroups.find(
+        (gp) => gp.groupId === populationCriteria.id
+      );
+      const results = selectedGroup?.populationValues
+        .filter((population) => {
+          return !population.name.includes("Observation");
+        })
+        .map((population, index) => {
+          return {
+            id: population.id,
+            criteriaReference: population.criteriaReference,
+            name: population.name,
+            abbreviation: getPopulationAbbreviation(
+              selectedGroup.populationValues,
+              population.name,
+              index
+            ),
+          };
+        });
+      setRelevantPops(results);
     }
-  };
-
-  const filterBasedOnDefinitionCategories = (definitionType) => {
-    const definitionNames = [
-      ...new Set(
-        Object.values(calculationOutput)?.flatMap((testCases) =>
-          Object.values(testCases)?.flatMap((testCaseCalculationResult) =>
-            Object.values(testCaseCalculationResult.statement_results)?.flatMap(
-              (statementResult) =>
-                Object.keys(statementResult)?.filter((definitionName) =>
-                  getDefintionCategoryFilteringCondition(
-                    definitionType,
-                    statementResult,
-                    definitionName
-                  )
-                )
-            )
-          )
-        )
-      ),
-    ];
+  }, [testCaseGroups, populationCriteria.id, getPopulationAbbreviation]);
 
-    return definitionNames.reduce((acc, definitionName) => {
-      if (mappedCql.definitions[definitionName]) {
-        acc[definitionName] = mappedCql.definitions[definitionName];
+  const getStatementResultsInCategory = (statementResults, definition) => {
+    if (statementResults) {
+      if (definition === "Used") {
+        return statementResults.filter((s) => s.relevance !== "NA");
```

Comment:
population definitions like initial population, denominator, Numerator, etc should be ignored here. need to show only used definitions.